application.json

```diff
@@ -1445,6 +1498,180 @@
         }
         ],
       "permalink": "linkedin"
+    },
+    {
+      "description": "",
+      "icon": "marketing-analytics/social",
+      "name": "Bing Ads",
+      "pages": [
+        {
+          "name": "Overview",
+          "permalink": "overview",
+          "type": "dashboardCollection",
+          "dashboardCollection": {
+            "dashboards": [
+              {
+                "lookmlID": "marketing_analytics::bing_overview",
+                "name": "Default",
+                "permalink": ""
+              }
+            ]
+          },
+          "filters": [
+            {
+              "defaultValue": "28 day",
+              "field": "fact.period",
+              "name": "Time Frame",
+              "type": "TimeFrame",
+              "dashboardAlias": "Period"
+            },
+            {
+              "field": "fact.campaign_name",
+              "name": "Campaign",
+              "type": "Table",
+              "tableID": "bi_campaign"
+            },
+            {
+              "autoSuggest": {
+              "model": "marketing_analytics",
```

Comment:
can you fix the indentation here?

Reply:
I think I fixed all the lines that were out of alignment